cockroachcloud/serverless-movr.md

```diff
@@ -0,0 +1,125 @@
+get-the-cluster-connection-string---
+title: Estimate Serverless Usage with MovR
+summary: Run the MovR sample application to estimate RU usage in CockroachDB Serverless.
+toc: true
+referral_id: docs_serverless_movr
+docs_area: get_started
+---
+
+This tutorial shows you how to initialize and run the MovR workload on a {{ site.data.products.serverless }} cluster, then use the Request Unit consumption data to estimate a sufficient budget for the MovR workload.
+
+## Overview
+
+MovR is a fictional company that offers users a platform for sharing vehicles, like scooters, bicycles, and skateboards, in select cities across the United States and Europe. The MovR team wants to use {{ site.data.products.serverless }} to run their high-growth, variable workload. They decide to test out a sample workload to see how it will perform and how much it will cost to run.
+
+## Before you begin
+
+- Make sure you have already [installed CockroachDB](../{{site.versions["stable"]}}/install-cockroachdb.html).
+- [Create a {{ site.data.products.serverless }} cluster](quickstart.html).
+
+## Step 1. Get the cluster connection string
+
+1. On the cluster's **Overview** page, click **Connect**.
+1. Select **General connection string** from the **Select option** dropdown.
+1. Open the **General connection string** section, then copy the connection string provided and save it in a secure location.
+
+## Step 2. Create the `movr` database
+
+1. Connect to the SQL shell using the connection string you copied in [Step 1](#step-1-get-the-cluster-connection-string):
+
+    {% include copy-clipboard.html %}
+    ~~~ shell
+    cockroach sql --url \
+    'postgresql://<username>:<password>@serverless-host>:26257/defaultdb?sslmode=verify-full&options=--cluster%3D<routing-id>'
+    ~~~
+    
+    Where:
+      - `<username>` is the SQL user. By default, this is your {{ site.data.products.db }} account username.
+      - `<password>` is the password for the SQL user. The password will be shown only once in the **Connect to cluster** dialog after creating the cluster.
+      - `<serverless-host>` is the hostname of the {{ site.data.products.serverless }} cluster.
+      - `<routing-id>` identifies your tenant cluster on a [multi-tenant host](https://www.cockroachlabs.com/docs/cockroachcloud/architecture.html#architecture). For example, `funny-skunk-123`.
```

Comment:
is there a way to link to the cloud arch doc using something like `architecture.html#architecture` instead of the full URL?

Reply:
yes, this was a copy-paste mistake. fixing the source as well.